trade-finance-manager-ui/styles/table.scss

```diff
@@ -156,4 +156,26 @@ table.premium-payments-table {
   .payments-list-item {
     list-style-type: none;
   }
-}
\ No newline at end of file
+}
+
+table.utilisation-report-reconciliation-table {
+  table-layout: fixed;
+
+  .ukef-width-one-tenth {
+    width: 10% !important;
+  }
+
+  .ukef-width-one-quarter {
+    width: 25% !important;
```

Comment:
Do we really need `!important`, I suspect so so that it overrides a design library class?

Reply:
Removed and retested - works as it did previously. Thanks for mentioning

(I think this was only required for a previous approach)